Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
```

Comment:
@fionatagious why are there so many dependency updates? I would expect there to be only -faker, ++faker, and updates to ffaker's dependencies.

Reply:
Hm, I don't know... maybe we can test this Gemfile.lock using your machine?